### DISCUSSION TITLE
Replacing Jetson Nano with Raspberry Pi 3 Model B+

### DISCUSSION THREAD

**Discussion by ekaramustafa** [prl-mushr/mushr#49] [Hardware]
Hi!

Would it be possible to do all steps and build the car if we replace Jetson Nano with Raspberry Pi 3 Model B+? Is there any alternative SBC for Jetson Nano?

**Comment by palanc** [comment]
Hi @ekaramustafa - We only support the Jetson Nano. In order to use the Raspberry Pi, you would have to make a fair number of changes to both the software and hardware. Let us know if you have any other questions!
